docs/cis_self_assessment.md

```diff
@@ -1648,32 +1651,19 @@ Ensure that a minimal audit policy is created (Scored)
 Logging is an important detective control for all systems, to detect potential unauthorised access.
 </details>
 
-**Result:** Pass
-
-**Remediation:**
-Create an audit policy file for your cluster.
-
-**Audit Script:** 3.2.1.sh
-
-```
-#!/bin/bash -e
-
-api_server_bin=${1}
-
-/bin/ps -ef | /bin/grep ${api_server_bin} | /bin/grep -v ${0} | /bin/grep -v grep
-```
+**Result:** FAIL TODO!!!
```

Comment:
```suggestion
**Result:** Scored - Operator Dependent
```
I'll give you something to tweak the remediation to explain

Reply:
Oh! I know what we can do: we can add audit configuration to the hardening guide.

Reply:
so, lets leave this as FAIL TODO until we have the hardening guide updated